src/profile.ts

```diff
@@ -10,9 +10,11 @@ export interface IProfile {
 
 export class Profile {
   profile: IProfile;
+  address: string;
 
-  constructor(profile: IProfile) {
+  constructor(profile: IProfile, address: string) {
```

Comment:
This is a solution to the issue that when creating multiple Orgs Profiles I don't have access to their initial address only their Profile.
So know the address forms part of the profile class instance